frame/staking/src/mock.rs

```diff
@@ -63,23 +59,14 @@ impl OneSessionHandler<AccountId> for OtherSessionHandler {
 	{
 	}
 
-	fn on_new_session<'a, I: 'a>(_: bool, validators: I, _: I)
+	fn on_new_session<'a, I: 'a>(_: bool, _validators: I, _: I)
```

Comment:
```suggestion
	fn on_new_session<'a, I: 'a>(_: bool, _: I, _: I)
```

for consistency with the other arguments